test/lib.sh

```diff
@@ -40,7 +40,8 @@ function start_knative_gcp_monitoring() {
 
 # Install all required components for running knative-gcp.
 function start_knative_gcp() {
-  start_latest_knative_serving || return 1
+  # Try reapply serving yaml again in case of race condition between webhook and config map
+  start_latest_knative_serving || start_latest_knative_serving || return 1
```

Comment:
Wow, we are really desperate.

Reply:
We are doing that for eventing too before the webhook is ignored.